src/tests/work-item/work-item-list/page-objects/dashboard.page.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * AlMighty page object example module for work item list page
+ * See: http://martinfowler.com/bliki/PageObject.html,
+ * https://www.thoughtworks.com/insights/blog/using-page-objects-overcome-protractors-shortcomings
+ * @author naverma@redhat.com
+ * TODO - Complete the page object model pending completion of UI at: http://demo.almighty.io/
+ */
+
+'use strict';
+
+/*
+ * Code Page Definition - placeholder
+ */
+
+let testSupport = require('../testSupport');
+let CommonPage = require('./common.page');
+let constants = require("../constants");
+let until = protractor.ExpectedConditions;
+
+class DashboardPage {
+
+ constructor(login) {
+ };
+ welcomeWrapper () {
+    return element(by.css("welcomeWrapper"));
```

Comment:
Each of the UI elements that you define here should also have a "click" function - are they currently not click-able?